### PR TITLE
fixing OCP and OpenShift Container Platform references

### PR DESCRIPTION
**Pull request by kalexand-rh**
This PR removes some incorrect references to OCP, Openshift, OpenShift, and OpenShift Container Platform. It also brings a few references to `oc` closer to the language in the [glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#openshift-cli).

4.8+

### CODE REVIEW

File: modules/gitops-synchronizing-your-application-application-with-your-git-repository.adoc [hunk 1]
```diff
@@ -6,15 +6,14 @@
...
+. In the Argo CD dashboard, notice that the *cluster-configs* Argo CD application has the statuses *Missing* and *OutOfSync*. Because the application was configured with a manual sync policy, Argo CD does not sync it automatically.
 
 . Click the *Sync* button on the *cluster-configs* tile, review the changes, and then click *Synchronize*. Argo CD will detect any changes in the Git repository automatically. If the configurations are changed, Argo CD will change the status of the *cluster-configs* to *OutOfSync*. You can modify the synchronization policy for Argo CD to automatically apply changes from your Git repository to the cluster.
 
 . Notice that the *cluster-configs* Argo CD application now has the statuses *Healthy* and *Synced*. Click the *cluster-configs* tile to check the details of the synchronized resources and their status on the cluster.
 
-. Navigate to the OpenShift web console and click {rh-app-icon} to verify that a link to the *Red Hat Developer Blog* is now present there. 
+. Navigate to the {product-title} web console and click {rh-app-icon} to verify that a link to the *Red Hat Developer Blog* is now present there.
```
Comment (bobfuru) [inline modules/gitops-synchronizing-your-application-application-with-your-git-repository.adoc:15]: Same comment about web console

File: modules/k8s-nmstate-installing-the-kubernetes-nmstate-operator.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 = Installing the Kubernetes NMState Operator
 
-You must install the Kubernetes NMState Operator from the OpenShift web console while logged in with administrator privileges. Once installed, the Operator can deploy the NMState State Controller as a daemon set across all of the cluster nodes.
+You must install the Kubernetes NMState Operator from the {product-title} web console while logged in with administrator privileges. Once installed, the Operator can deploy the NMState State Controller as a daemon set across all of the cluster nodes.
```
Comment (bobfuru) [inline modules/k8s-nmstate-installing-the-kubernetes-nmstate-operator.adoc:9]: Same comment about web console

File: modules/kube-apiserver-operator.adoc [hunk 3]
```diff
@@ -8,7 +8,7 @@
 [discrete]
 == Purpose
 
-The Kubernetes API Server Operator manages and updates the Kubernetes API server deployed on top of {product-title}. The Operator is based on the OpenShift library-go framework and it is installed using the Cluster Version Operator (CVO).
+The Kubernetes API Server Operator manages and updates the Kubernetes API server deployed on top of {product-title}. The Operator is based on the {product-title} library-go framework and it is installed using the Cluster Version Operator (CVO).
```
Comment (bobfuru) [inline modules/kube-apiserver-operator.adoc:11]: Out of scope but I see in the next module that "library-go" is in backticks. 🤷

File: web_console/odc-about-web-terminal.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can launch an embedded command line terminal instance in the OpenShift web console. You must first install the Web Terminal Operator to use the web terminal.
+You can launch an embedded command line terminal instance in the {product-title} web console. You must first install the Web Terminal Operator to use the web terminal.
```
Comment (bobfuru) [inline web_console/odc-about-web-terminal.adoc:8]: Same comment about web console

**Review by bobfuru — COMMENTED** [review]
Overall, this LGTM. I added comments where you might consider using just "web console" rather than "{product-title} web console" but that is not what this PR is about, so your call. Thanks for doing this!!

File: modules/gitops-creating-an-application-by-using-the-argo-cd-dashboard.adoc [hunk 5]
```diff
@@ -16,7 +16,7 @@ endif::[]
 Argo CD provides a dashboard which allows you to create applications.
 
 ifdef::cluster[]
-This sample workflow walks you through the process of configuring Argo CD to recursively sync the content of the `cluster` directory to the `cluster-configs` application. The directory defines OpenShift web console cluster configurations that add a link to the *Red Hat Developer Blog* under the {rh-app-icon} menu in the web console, and defines a namespace `spring-petclinic` on the cluster.
+This sample workflow walks you through the process of configuring Argo CD to recursively sync the content of the `cluster` directory to the `cluster-configs` application. The directory defines the {product-title} web console cluster configurations that add a link to the *Red Hat Developer Blog* under the {rh-app-icon} menu in the web console, and defines a namespace `spring-petclinic` on the cluster.
```
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply modules/gitops-creating-an-application-by-using-the-argo-cd-dashboard.adoc:19]: I'm going to leave the ones for Argo CD for clarity. Argo CD has its own dashboard.

**Pull request merged**